crates/rust-analyzer/src/reload.rs

```diff
@@ -503,7 +503,7 @@ impl GlobalState {
             let mut crate_graph_file_dependencies = FxHashSet::default();
 
             let mut load = |path: &AbsPath| {
-                let _p = tracing::span!(tracing::Level::INFO, "switch_workspaces::load").entered();
+                let _p = tracing::span!(tracing::Level::DEBUG, "switch_workspaces::load").entered();
```

Comment:
it's up to you, but this change means that `hprof` won't pick up on this span.

Reply:
My base logs became completely useless due to that trace hence the change. Should be fine not to have that included in hprof